scripts/website/generate.go

```diff
@@ -0,0 +1,206 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"gopkg.in/yaml.v3"
+)
+
+// Suite ...
+// ToDo (nilekh): Get this struct from the Gatekeeper repo
+type Suite struct {
+	Kind       string `yaml:"kind"`
+	APIVersion string `yaml:"apiVersion"`
+	Metadata   struct {
+		Name string `yaml:"name"`
+	} `yaml:"metadata"`
+	Tests []struct {
+		Name       string `yaml:"name"`
+		Template   string `yaml:"template"`
+		Constraint string `yaml:"constraint"`
+		Cases      []struct {
+			Name       string `yaml:"name"`
+			Object     string `yaml:"object"`
+			Assertions []struct {
+				Violations string `yaml:"violations"`
+			} `yaml:"assertions"`
+		} `yaml:"cases"`
+	} `yaml:"tests"`
+}
+
+// ConstraintTemplate ...
+// ToDo (nilekh): Get this struct from the Gatekeeper repo
```

Comment:
framework repo has the CT structs

Reply:
I had looked at it, but they lack yaml annotations. Am I missing anything here?

Reply:
Switched to unstructured reading.